frontend/public/models/index.ts

```diff
@@ -1360,3 +1360,19 @@ export const ConsolePluginModel: K8sKind = {
   labelPluralKey: 'public~ConsolePlugins',
   crd: true,
 };
+
+export const PodDisruptionBudgetModel: K8sKind = {
+  label: 'PodDisruptionBudget',
+  // t('public~PodDisruptionBudget')
+  labelKey: 'public~PodDisruptionBudget',
+  labelPlural: 'PodDisruptionBudgets',
+  // t('public~PodDisruptionBudgets')
+  labelPluralKey: 'public~PodDisruptionBudgets',
+  plural: 'poddisruptionbudgets',
+  apiVersion: 'v1',
+  apiGroup: 'policy',
```

Comment:
Is this correct? Normally an API group is a domain name.

Reply:
Yup, comes from the K8s documentation too
```
apiVersion: policy/v1
kind: PodDisruptionBudget
metadata:
  name: zk-pdb
spec:
  minAvailable: 2
  selector:
    matchLabels:
      app: zookeeper
```
Ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/